### PR TITLE
Add metalsmith plugins

### PR DESCRIPTION
**Pull request by fortes**
Added

* metalsmith-alias
* metalsmith-code-highlight
* metalsmith-jekyll-dates

### CODE REVIEW

File: src/plugins.json [hunk 1]
```diff
@@ -1,5 +1,10 @@
 [
   {
+    "name": "Alias",
+    "icon": "link",
+    "repository": "https://github.com/fortes/metalsmith-alias",
+    "description": "Create URL aliases for any page."
+  },{
```
Comment (Ajedi32) [inline src/plugins.json]: Minor whitespace issue here

File: src/plugins.json [hunk 2]
```diff
@@ -84,6 +89,12 @@
     "description": "Compile CoffeeScript files to Javascript."
   },
   {
+     "name": "Code Highlight",
```
Comment (Ajedi32) [inline src/plugins.json]: Shouldn't "**Cod**e Highlight" come before "**Col**lections"?

File: src/plugins.json (hunk 1 above)
Comment (fortes) [inline src/plugins.json]: Fixed

File: src/plugins.json (hunk 2 above)
Comment (fortes) [inline src/plugins.json]: Fixed. Turns out the Coffee plugin was out of order too

**Pull request merged**